lib/disc.rb

```diff
@@ -110,10 +119,14 @@ def disc_options
       end
 
       def queue
-        disc_options.fetch(:queue, 'default')
+        @queue || Disc.default_queue
```

Comment:
I agree it's not *supposed* to be called multiple times. But there's no reason to make this hide state like that :confused:

I'd say this is a bug because this would be **completely** unexpected when you accidentally stumble into this in production ("Why the :boom: are these jobs being enqueued in the default queue instead of the urgent queue?").

This is more along the lines of what I had in mind: https://github.com/pote/disc/compare/extend-job-options

WDYT?